website/docs/compose/compose-spec.md

```diff
@@ -5,18 +5,24 @@ sidebar_position: 1
 
 ## Introduction
 
-Podman Desktop supports the [Compose spec](https://compose-spec.io). With Podman Desktop, users can easily create and manage multi-container applications by using Compose files.
+Podman Desktop supports the [Compose specification](https://compose-spec.io). With Podman Desktop, users can easily create and manage multi-container applications by using Compose files.
+
+We support two version of the Compose specification:
```

Comment:
This line conflicts with the last line ("Any containers already deployed by Docker Compose / Podman Compose will be automatically shown within Podman Desktop. You do not need to do anything!").

I believe we have to distinguish:

* Podman Desktop automatically detects any container or service that has a valid compose label (regardless of the Compose engine that created the container or service).

* Podman Destkop can install and run two Compose engines: Compose and Podman Compose.